fluidkeys/keycreate.go

```diff
@@ -178,6 +178,14 @@ func promptAndPublishToFluidkeysDirectory(privateKey *pgpkey.PgpKey) {
 }
 
 func tryToPublishKeyAndSetAllowSearchByEmail(privateKey *pgpkey.PgpKey) error {
+	if privateKey.PrivateKey == nil {
```

Comment:
Excellent spot. I'd have no objection to combining these into 1:
```
if privateKey.PrivateKey == nil || privateKey.PrivateKey.Encrypted {
```

whaddya think?